skimage/io/_io.py

```diff
@@ -124,6 +124,13 @@ def imsave(fname, arr, plugin=None, **plugin_args):
     plugin_args : keywords
         Passed to the given plugin.
 
+    Notes
+    -----
+    When saving a JPEG, the compression ratio may be controlled using the
+    ``quality`` keyword argument which is an integer with values in [1, 100]
+    where 1 is worst quality and smallest file size, and 100 is best quality and
+    largest file size (default 75).  This is only available when using the PIL
```

Comment:
@ahojnnes I bet we should care too much about FreeImage - https://github.com/scikit-image/scikit-image/pull/1933 .